### PR TITLE
GO-2828: add debug diagnostics for spaces and connections

### PR DESCRIPTION
**Pull request by AnastasiaShemyakinskaya**
https://linear.app/anytype/issue/GO-2828/add-debug-diagnostics-for-spaces-and-connections

### CODE REVIEW

File: net/pool/pool_test.go [hunk 1]
```diff
@@ -240,16 +357,31 @@ func (d *dialerMock) Name() (name string) {
 	return "net.peerservice"
 }
 
-func newTestPeer(id string) *testPeer {
+func newTestPeer(id string, created time.Time, subConnections int, version uint32) *testPeer {
```
Comment (mcrakhman) [inline net/pool/pool_test.go:360]: make another constructor for testPeer with all that stuff, don't change other tests

File: net/peer/peer.go [hunk 2]
```diff
@@ -320,3 +331,16 @@ func (p *peer) Close() (err error) {
...
+
+func (p *peer) ProvideStat() *Stat {
+	protoVersion, _ := CtxProtoVersion(p.Context())
+	p.mu.Lock()
+	subConnectionsCount := len(p.active)
+	p.mu.Unlock()
+	return &Stat{
+		PeerId:         p.id,
+		SubConnections: subConnectionsCount,
+		Created:        p.created,
```
Comment (mcrakhman) [inline net/peer/peer.go:343]: add a field called `AliveTimeSecs` which will tell how many seconds it is alive

Comment (mcrakhman) [reply net/peer/peer.go:343]: Just write pls p.Lock(), defer p.Unlock() and then return &Stat{...}

Comment (AnastasiaShemyakinskaya) [reply net/peer/peer.go:343]: How do you propose to calculate `AliveTimeSecs`, because don't see such field in peer and? 🧐

Comment (AnastasiaShemyakinskaya) [reply net/peer/peer.go:344]: Fixed

**Review by mcrakhman — APPROVED** [review]

**Pull request merged**